static-sites.yml

```diff
@@ -24,7 +24,7 @@ all:
   cinescuela.es:
     name: Cinescuela
     version: 2016-10-20
-    description: Lleva el cine a tu aula de clase.
+    description: Lleva el cine a tu biblioteca móvil.
```

Comment:
May be better yes. But this description change has been asked by a person from the Biblioteca nacional here, so let me check this with her tomorrow, not to change that behind his back :)